starknet_py/net/schemas/rpc.py

```diff
@@ -253,50 +256,78 @@ class TransactionV3Schema(TransactionSchema):
     )
 
 
-class DeprecatedInvokeTransactionSchema(DeprecatedTransactionSchema):
-    contract_address = Felt(data_key="contract_address", load_default=None)
-    sender_address = Felt(data_key="sender_address", load_default=None)
-    entry_point_selector = Felt(data_key="entry_point_selector", load_default=None)
+class InvokeTransactionV0Schema(DeprecatedTransactionSchema):
     calldata = fields.List(Felt(), data_key="calldata", required=True)
-    nonce = Felt(data_key="nonce", load_default=None)
+    contract_address = Felt(data_key="contract_address", required=True)
+    entry_point_selector = Felt(data_key="entry_point_selector", required=True)
 
     @post_load
-    def make_transaction(self, data, **kwargs) -> InvokeTransaction:
+    def make_transaction(self, data, **kwargs) -> InvokeTransactionV0:
         _replace_invoke_contract_address_with_sender_address(data)
```

Comment:
This function call can now be removed, and the function itself can also be removed.